### PR TITLE
add port to connection string

### LINKED ISSUE #6: Migration not working on postgres database with non default port
Works fine with default port, not when other port is specified.
localsetup.py
```
'mara': mara_db.dbs.PostgreSQLDB(user='user', password='pw', port=10000, host='localhost', database='database')
```

> make
```
...
migrate-mara-db: Could not access or create database "postgresql+psycopg2://user:pw@localhost/database":
migrate-mara-db: (psycopg2.ProgrammingError) permission denied to create database
...
```

### PR DESCRIPTION
**Pull request by Tafkas**
This PR adds the port to the connection string allowing non-default ports for postgres. Fixes #6

### CODE REVIEW

File: mara_db/auto_migration.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def __(db, **_):
 @engine.register(mara_db.dbs.PostgreSQLDB)
 def __(db: mara_db.dbs.PostgreSQLDB):
     return sqlalchemy.create_engine(
-        f'postgresql+psycopg2://{db.user}{":"+db.password if db.password else ""}@{db.host}/{db.database}')
+        f'postgresql+psycopg2://{db.user}{":"+db.password if db.password else ""}@{db.host}":"{db.port}/{db.database}')
```
Comment (alexandschaefer) [inline mara_db/auto_migration.py]: throwes an error like this: 

```([lmw-dwh/.venv]) alexander@PAV-NB-PC08BA8HF:~/projects/lampenwelt/lmw-dwh(change-BQ-session-def)$ flask mara_db.migrate
Traceback (most recent call last):
  File "/home/alexander/projects/lampenwelt/lmw-dwh/.venv/bin/flask", line 11, in <module>
    sys.exit(main())
  File "/home/alexander/projects/lampenwelt/lmw-dwh/.venv/lib/python3.6/site-packages/flask/cli.py", line 513, in main
    cli.main(args=args, prog_name=name)
  File "/home/alexander/projects/lampenwelt/lmw-dwh/.venv/lib/python3.6/site-packages/flask/cli.py", line 380, in main
    return AppGroup.main(self, *args, **kwargs)
  File "/home/alexander/projects/lampenwelt/lmw-dwh/.venv/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/alexander/projects/lampenwelt/lmw-dwh/.venv/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/alexander/projects/lampenwelt/lmw-dwh/.venv/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/alexander/projects/lampenwelt/lmw-dwh/.venv/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/alexander/projects/lampenwelt/lmw-dwh/packages/mara-db/mara_db/cli.py", line 13, in migrate
    if not mara_db.auto_migration.auto_discover_models_and_migrate():
  File "/home/alexander/projects/lampenwelt/lmw-dwh/packages/mara-db/mara_db/auto_migration.py", line 120, in auto_discover_models_and_migrate
    return auto_migrate(engine('mara'), models)
  File "/usr/lib/python3.6/functools.py", line 803, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
  File "/home/alexander/projects/lampenwelt/lmw-dwh/packages/mara-db/mara_db/auto_migration.py", line 143, in __
    return engine(mara_db.dbs.db(alias))
  File "/usr/lib/python3.6/functools.py", line 803, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
  File "/home/alexander/projects/lampenwelt/lmw-dwh/packages/mara-db/mara_db/auto_migration.py", line 154, in __
    f'postgresql+psycopg2://{db.user}{":"+db.password if db.password else ""}@{db.host}":"{db.port}/{db.database}')
  File "/home/alexander/projects/lampenwelt/lmw-dwh/.venv/lib/python3.6/site-packages/sqlalchemy/engine/__init__.py", line 424, in create_engine
    return strategy.create(*args, **kwargs)
  File "/home/alexander/projects/lampenwelt/lmw-dwh/.venv/lib/python3.6/site-packages/sqlalchemy/engine/strategies.py", line 50, in create
    u = url.make_url(name_or_url)
  File "/home/alexander/projects/lampenwelt/lmw-dwh/.venv/lib/python3.6/site-packages/sqlalchemy/engine/url.py", line 211, in make_url
    return _parse_rfc1738_args(name_or_url)
  File "/home/alexander/projects/lampenwelt/lmw-dwh/.venv/lib/python3.6/site-packages/sqlalchemy/engine/url.py", line 267, in _parse_rfc1738_args
    return URL(name, **components)
  File "/home/alexander/projects/lampenwelt/lmw-dwh/.venv/lib/python3.6/site-packages/sqlalchemy/engine/url.py", line 60, in __init__
    self.port = int(port)
ValueError: invalid literal for int() with base 10: '"None ```

Comment (alexandschaefer) [reply mara_db/auto_migration.py]: Would this fix having a different port in general or only the migration part?

Comment (jankatins) [inline mara_db/auto_migration.py]: O simple 

```python
f'postgresql+psycopg2://{db.user}{":"+db.password if db.password else ""}@{db.host}{":"+db.port if db.port else "" }/{db.database}')
```

Comment (martin-loetzsch) [reply mara_db/auto_migration.py]: yep

**Pull request merged**